testnet/wallet-options-v4.json

```diff
@@ -2,28 +2,39 @@
   "platforms": {
     "web": {
       "application": {
+        "analyticsSiteId": 2,
         "enableDomainMigrationRedirects": true,
         "announcements": {
           "public": {
-            "id": "testnet-warning-public",
+            "id": "testnet-warning",
             "type": "danger",
             "icon": "alert-filled",
             "header": {
-              "en": "Warning! This is the testnet3 blockchain. Testnet coins have no value."
-            }
+              "en": "TESTNET WARNING"
+            },
+            "sections": [
+              {
+                "body": {
+                  "en": "This is the testnet3 blockchain. Your coins have no value here."
```

Comment:
I'd prefer this language: "Warning! This is the testnet3 blockchain. Testnet coins have no value."

Reply:
👍